### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#83] [Ideas]
An enhancement for this section would be:  Add Border    Style   Thickness   Colour to the image selected

Possible also an option to add a caption for the image.

As for stop and piston images, it would be nice to be able to have text positioned differently between the off and on images to simulate text moving along with the stop or piston. 

This is possible in jOrgan but not in GrandOrgue (apologies for the comparison but it highlights the fact that it is possible). It would eliminate the need for the use of multiple, already labeled, individual, separate on/off images for each element. 

Mark

**Comment by larspalo** [comment]
All of the suggestions above should (separately when unrelated things are mentioned - like general image stuff and stop on/off images) be raised as issues for GrandOrgue at https://github.com/GrandOrgue/grandorgue/issues with the label of enhancement.

> It would eliminate the need for the use of multiple, already labeled, individual, separate on/off images for each element.

Personally I think that I'll continue to prefer creating custom versions myself for the organ I'm modeling rather than using any generic built in versions even if it would be available. But maybe someone thinks that the idea could be useful enough for implementation.
